targets/targets.json

```diff
@@ -1349,7 +1349,7 @@
         "core": "Cortex-M4F",
         "extra_labels_add": ["STM32L4", "STM32L476xG", "STM32L476VG"],
         "detect_code": ["0820"],
-        "macros_add": ["USBHOST_OTHER"],
+        "macros_add": ["USBHOST_OTHER", "PAL_USE_HW_ROT=1"],
         "device_has_add": ["ANALOGOUT", "CAN", "LOWPOWERTIMER", "SERIAL_FC", "TRNG", "FLASH"],
```

Comment:
@MarceloSalazar Should I move the macro definition into mbed-cloud-client-example-sources-internal\mbed-cloud-client\mbed_lib.json or mbed-cloud-client\mbed-client-pal\mbed_lib.json? I think, F429 also require it.